repository/zookeeper/operator/operator.yaml

```diff
@@ -39,14 +50,15 @@ plans:
           - name: validation
             tasks:
               - validation
-            delete: true
   validation:
     strategy: serial
     phases:
       - name: connection
-        strategy: parallel
+        strategy: serial
         steps:
           - name: connection
             tasks:
               - validation
-            delete: true
```

Comment:
Fun fact: this `delete: true` was a bug: effectively this plan was deleting a non-existing job (`validation.yaml`) and was immediately successful. So no validation happened. This is now fixed on the right side /cc @gerred @zmalik

Reply:
nice @zen-dog - that's awesome.